parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -419,7 +423,7 @@ systemctlEnableAndStart() {
         echo "$1 could not be started"
         return 1
     fi
-    if ! retrycmd_if_failure 120 5 25 systemctl enable $1; then
+    if ! retrycmd 120 5 25 systemctl enable $1; then
```

Comment:
I can't find where `retrycmd` is defined. typo ?

Reply:
Thanks, missed this - was playing around with renaming earlier but decided to keep most names as is